templates/terraform/examples/backend_service_network_endpoint.tf.erb

```diff
@@ -0,0 +1,24 @@
+resource "google_compute_global_network_endpoint_group" "external_proxy" {
+  name                  = "<%= ctx[:vars]['neg_name'] %>"
+  network_endpoint_type = "INTERNET_FQDN_PORT"
+  default_port          = "443"
+}
+
+resource "google_compute_global_network_endpoint" "proxy" {
+  global_network_endpoint_group = google_compute_global_network_endpoint_group.external_proxy.name
```

Comment:
can this be `.id`? (https://github.com/terraform-providers/terraform-provider-google/wiki/Developer-Best-Practices#self-links-names-and-ids)